### PR TITLE
Workflow tweak - add model name to comment

### LINKED ISSUE #45: Evaluation stage
- Evaluation-stage. Output:
    -  [x] Confusion matrix (png)
    - [x] metrics.txt
    - [x] Optional: plots of loss, accuracy and f1_macro per epoch

### LINKED ISSUE #68: Add model name to iterative report
- [ ] In den Kommentären von [cdl1-iterative](https://github.com/cdl1-iterative) soll der Modellname erscheinen (im Titel).

### PR DESCRIPTION
**Pull request by gabrieltorresgamez**

### CODE REVIEW

**Comment by ghost** [comment]
# RUN: HistGradientBoostingClassifier, 2023-05-01 09-31-15

## Hyperparameters

| Path        | Param                               | HEAD^   | workspace   |
|-------------|-------------------------------------|---------|-------------|
| params.yaml | dvclive.model_hparams.learning_rate | 0.15    | 0.1         |

## Metrics

| Path   | Metric   | HEAD^   | workspace   | Change   |
|--------|----------|---------|-------------|----------|

## Plots

### Confusion Matrix

Confusion Matrix

**Comment by gabrieltorresgamez** [comment]
there are some problem with the output from last stage, don't merge yet

**Comment by ghost** [comment]
# RUN: HistGradientBoostingClassifier, 2023-05-01 11-15-13

## Hyperparameters

| Path   | Param   | main   | workspace   |
|--------|---------|--------|-------------|

## Metrics

| Path   | Metric   | main   | workspace   | Change   |
|--------|----------|--------|-------------|----------|

## Plots

### Confusion Matrix

Confusion Matrix

**Comment by ghost** [comment]
# RUN: HistGradientBoostingClassifier, 2023-05-01 11-57-52

## Hyperparameters

| Path        | Param                               | main   | workspace   |
|-------------|-------------------------------------|--------|-------------|
| params.yaml | dvclive.model_hparams.learning_rate | 0.1    | 0.09        |

## Metrics

| Path                 | Metric         | main    | workspace   | Change   |
|----------------------|----------------|---------|-------------|----------|
| dvclive/metrics.json | test.epoch.acc | 0.37634 | 0.38996     | 0.01362  |
| dvclive/metrics.json | test.epoch.f1  | 0.29114 | 0.31019     | 0.01904  |

## Plots

### Confusion Matrix

Confusion Matrix

**Comment by gabrieltorresgamez** [comment]
added model name to comment and converted every output model to a pickle
please review @florinbarbisch and @yvokeller :)

**Review by florinbarbisch — APPROVED** [review]

**Comment by ghost** [comment]
# RUN: HistGradientBoostingClassifier, 2023-05-01 12-19-02

## Hyperparameters

| Path        | Param                               | main   | workspace   |
|-------------|-------------------------------------|--------|-------------|
| params.yaml | dvclive.model_hparams.learning_rate | 0.1    | 0.2         |

## Metrics

| Path                 | Metric         | main    | workspace   | Change   |
|----------------------|----------------|---------|-------------|----------|
| dvclive/metrics.json | test.epoch.acc | 0.37634 | 0.39211     | 0.01577  |
| dvclive/metrics.json | test.epoch.f1  | 0.29114 | 0.31129     | 0.02015  |

## Plots

### Confusion Matrix

Confusion Matrix

**Review by florinbarbisch — APPROVED** [review]
nice

**Review by yvokeller — APPROVED** [review]
great 👍🏻
